.github/workflows/build.yml

```diff
@@ -379,19 +382,13 @@ jobs:
           name: ${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_linux_${{ matrix.arch }}.zip
 
       # build docker image
-      - name: Docker build
-        working-directory: ./.release/docker
-        run: |
-          docker version
-          unzip -j ${GITHUB_WORKSPACE}/${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_linux_${{ matrix.arch }}.zip
-          # arm32 bit docker images are set with --platform=linux/arm/v6
-          if [ "${{ matrix.arch }}" = "arm" ]; then
-            make docker ARCH=${{ matrix.arch }} GOARM_DOCKER="/v6"
-          else
-            make docker ARCH=${{ matrix.arch }}
-          fi
-      # upload image tarball
-      - uses: actions/upload-artifact@v2
+      - name: Docker build (Action)
+        uses: hashicorp/actions-docker-build@v1
         with:
-          name: ${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_docker_linux_${{ matrix.arch }}.tar
-          path: .release/docker/${{ env.PKG_NAME }}_${{ needs.get-product-version.outputs.product-version }}_docker_linux_${{ matrix.arch }}.tar
+          version: ${{env.version}}
+          target: default
+          dockerfile: .release/docker/Dockerfile
```

Comment:
If possible, we should move the Dockerfile to the default location (the root of the repo), so that we don't need to specify this config value. This has the other benefit that file paths in the Dockerfile then become relative to the Dockerfile's location which seems more intuitive.

Reply:
+1 on this for consistency across products

Reply:
@samsalisbury and @kpenfound, I moved the Dockerfile to the root of the repo